NavigationCodelab/app/src/main/java/com/example/compose/rally/ui/accounts/AccountsScreen.kt

```diff
@@ -23,24 +23,23 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.semantics.contentDescription
 import androidx.compose.ui.semantics.semantics
 import com.example.compose.rally.R
-import com.example.compose.rally.data.Account
+import com.example.compose.rally.data.UserData
 import com.example.compose.rally.ui.components.AccountRow
 import com.example.compose.rally.ui.components.StatementBody
 
 /**
  * The Accounts screen.
  */
 @Composable
-fun AccountsBody(
-    accounts: List<Account>,
+fun AccountsScreen(
     onAccountClick: (String) -> Unit = {},
 ) {
     StatementBody(
         modifier = Modifier.semantics { contentDescription = "Accounts Screen" },
-        items = accounts,
+        items = UserData.accounts,
         amounts = { account -> account.balance },
         colors = { account -> account.color },
-        amountsTotal = accounts.map { account -> account.balance }.sum(),
+        amountsTotal = UserData.accounts.map { account -> account.balance }.sum(),
```

Comment:
Nit: is it worth remembering this calculation instead of mapping and calling `sum` on every recomposition?